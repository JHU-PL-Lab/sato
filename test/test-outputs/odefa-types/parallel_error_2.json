[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "r", "a" ],
          "err_match_val": "input",
          "err_match_expected": "{lbl}",
          "err_match_actual": "int"
        }
      ]
    ],
    "err_input_seq": [ 0 ],
    "err_location": "p = r.lbl",
    "err_steps": 352
  }
]