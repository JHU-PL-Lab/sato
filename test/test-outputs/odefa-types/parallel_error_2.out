Type errors for:
- Input sequence  : [0]
- Found at clause : p = r.lbl
--------------------
* Value    : r = a = input
* Expected : {lbl}
* Actual   : int
1 errors found. No further control flows exist.
